src/bosh-director/lib/bosh/director/metadata_updater.rb

```diff
@@ -19,6 +19,7 @@ def update_vm_metadata(instance, metadata, factory = CloudFactory.create_with_la
         metadata['deployment'] = instance.deployment.name
         metadata['id'] = instance.uuid
```

Comment:
Well reasoned. Personally I consider tags to be global and shared across resources if they have the same meaning/context, so I generally lean towards consistency between the two resources. Optimistically, I think these tags would be namespaced with `bosh.` or something as well, but I'm not holding my breath for that idea :)

But you're right, these tags are definitely within the context of an instance, so they could theoretically be just `group`. Given current conventions,`group` might be more appropriate. I defer to @cppforlife.